radio/src/gui/colorlcd/radio_hardware.cpp

```diff
@@ -235,6 +235,20 @@ class SerialConfigWindow : public FormGroup
       aux->setAvailableHandler(
           [=](int value) { return isSerialModeAvailable(port_nr, value); });
       grid.nextLine();
+
+      if (port_nr < SP_VCP)
+      {
+          new StaticText(this, grid.getLabelSlot(), "Port power:", 0, COLOR_THEME_PRIMARY1);
```

Comment:
grid.getLabelSlot(true) will indent the text ;)

Reply:
Thanks, updated!